src/cli/cmds/synth.ts

```diff
@@ -75,4 +116,129 @@ async function fetchValidations(): Promise<ValidationConfig[] | undefined> {
 }
 
 
+async function helmSynthesis(apiVersion: string, chartVersion: string, outdir: string) {
+  const TEMPLATE_WITHOUT_CRDS = 'helm-chart-without-crds';
+  const TEMPLATE_WITH_CRDS = 'helm-chart-with-crds';
+
+  const pkgroot = path.join(__dirname, '..', '..', '..');
+  const templatesDir = path.join(pkgroot, 'templates');
+
+  let templatePath: string;
+
+  const substituteValues = {
+    apiVersion: apiVersion,
+    version: chartVersion,
+    app: path.basename(path.resolve()),
+  };
+
+  // Helm chart structure without CRDs support
+  if (apiVersion === HelmChartApiVersion.V1) {
+
+    if (config.imports && config.imports.length > 0) {
```

Comment:
This is not the correct check because our normal `k8s` imports are also part of this, but they are not CRDs. Make sure we have a test for this scenario.

Reply:
Makes sense. Will test this and make updates.